app/src/main/java/com/github/sdp/ratemyepfl/database/GradeInfoRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.sdp.ratemyepfl.database
+
+import com.github.sdp.ratemyepfl.model.GradeInfo
+import com.github.sdp.ratemyepfl.model.review.ReviewRating
+import com.google.android.gms.tasks.Task
+import com.google.firebase.firestore.Transaction
+
+interface GradeInfoRepository {
+
+    fun update(id: String, transform: (GradeInfo) -> GradeInfo): Task<Transaction>
+
+    fun updateLikeRatio(id: String, rid: String, inc : Int): Task<Transaction>
```

Comment:
It may be clearer to understand if you write `itemId `and `reviewId`.

Reply:
Done